release-notes/release-2.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: February 13, 2024
+---
+
+# Features
+- PHP Azure Build Pipelines
+- [RUM Mobile - React Native](https://docs.middleware.io/rum/rum-react)
+- Spend Analysis charts
+- Trace correlation drill down from specific log messages
+- Price calculator that analyzes your bill based on the usage of different products
+- Sort and group attributes of log-collected data
+
+# Changes
+- Billing tab now displays information about plan name, billing date, billing address, payment details and option to update billing information
+- Invoice tab now displays all past invoices with invoice date, paid date, and the option to download invoices
+- New default dashboard for Kubernetes with pod and cluster overview metrics
+
+# Fixes
```

Comment:
We should be able to put a lot more in this list since there was a month between the last release and this one. Recommend working with Vivek to produce something more meaty.